src/home_robot_hw/home_robot_hw/stretch_client/modules/manip.py

```diff
@@ -0,0 +1,206 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+from typing import List, Optional
+
+import numpy as np
+import rospy
+from scipy.spatial.transform import Rotation as R
+from std_srvs.srv import TriggerRequest
+
+from home_robot.core.state import ManipulatorBaseParams
+from home_robot.motion.stretch import STRETCH_HOME_Q, HelloStretchIdx
+from home_robot.utils.geometry import posquat2sophus, xyt2sophus
+
+from .abstract import AbstractControlModule, enforce_enabled
+
+
+class StretchManipulationInterface(AbstractControlModule):
```

Comment:
maybe "Client" too instead of interface? intercface seems overloaded

Reply:
or "control"

Reply:
I agree that "Interface" seems loaded, but "Control" doesn't seem to encapsulate the part where we also query observations through it? Not sure. Renaming to client for now, happy to discuss if there are any concerns.